packages/app-project/src/shared/components/ProjectStatistics/ProjectStatisticsContainer.js

```diff
@@ -1,53 +1,51 @@
-import { inject, observer } from 'mobx-react'
+import { useContext } from 'react'
+import { MobXProviderContext, observer } from 'mobx-react'
 import PropTypes from 'prop-types'
-import { Component } from 'react'
-import { withRouter } from 'next/router'
-
+import { useRouter } from 'next/router'
 import ProjectStatistics from './ProjectStatistics'
 
-function storeMapper (stores) {
-  const { project } = stores.store
+function useStore () {
+  const { store } = useContext(MobXProviderContext)
+
   return {
-    classifications: project['classifications_count'],
-    completedSubjects: project['retired_subjects_count'],
-    projectName: project['display_name'],
-    subjects: project['subjects_count'],
-    volunteers: project['classifiers_count']
+    classifications: store?.project?.classifications_count,
+    completedSubjects: store?.project?.retired_subjects_count,
+    projectName: store?.project?.display_name,
+    subjects: store?.project?.subjects_count,
+    volunteers: store?.project?.classifiers_count
   }
 }
 
-class ProjectStatisticsContainer extends Component {
-  getLinkProps () {
-    const { router } = this.props
-    const owner = router?.query?.owner
-    const project = router?.query?.project
-    return {
-      href: `/projects/${owner}/${project}/stats`
-    }
+function getLinkProps ({ router }) {
+  const owner = router?.query?.owner
+  const project = router?.query?.project
+  return {
+    href: `/projects/${owner}/${project}/stats`
   }
+}
 
-  render () {
-    const {
-      className,
-      classifications,
-      completedSubjects,
-      projectName,
-      subjects,
-      volunteers
-    } = this.props
-
-    return (
-      <ProjectStatistics
-        className={className}
-        classifications={classifications}
-        completedSubjects={completedSubjects}
-        linkProps={this.getLinkProps()}
-        projectName={projectName}
-        subjects={subjects}
-        volunteers={volunteers}
-      />
-    )
-  }
+const ProjectStatisticsContainer = ({ className }) => {
+  const {
+    classifications,
+    completedSubjects,
+    projectName,
+    subjects,
+    volunteers
+  } = useStore()
+
+  const linkProps = getLinkProps({ router: useRouter() });
```

Comment:
```suggestion
  const router = useRouter()
  const owner = router?.query?.owner
  const project = router?.query?.project
  const linkProps = {
     href: `/projects/${owner}/${project}/stats`
  }
```

It feels really strange to call a React hook as the argument to a function. This component only needs an href for `linkProps`. I've suggested changes in this file to simplify things.